common/scripted_triggers/wc_culture_triggers.txt

```diff
@@ -276,7 +276,9 @@ has_horde_culture_trigger = {
 		}
 	}
 }
-
+has_grummle_culture_trigger = {
```

Comment:
Nah, I would rename it. The idea of the trigger is to see if the culture can unlock `tradition_himalayan_settlers` so the trigger should go to `common/scripted_triggers/wc_cultural_traditions_trigger.txt` and have an appropriate name.